### PR TITLE
Bump boto3 and botocore versions

### PR DESCRIPTION
**Pull request by alinabuzachis**
##### SUMMARY

Bump boto3 and botocore versions

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- New Module Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by alinabuzachis** [comment]
recheck

**Comment by alinabuzachis** [comment]
recheck

File: changelogs/fragments/bump_boto3_botocore_versions.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+major_changes:
+- amazon.cloud collection - The amazon.collection collection has dropped support for
```
Comment (gravesm) [inline changelogs/fragments/bump_boto3_botocore_versions.yml:2]:
```suggestion
- amazon.cloud collection - The amazon.cloud collection has dropped support for
```

File: requirements.txt [hunk 2]
```diff
@@ -2,6 +2,6 @@
 # - tests/unit/constraints.txt
 # - tests/integration/constraints.txt
 # - tests/integration/targets/setup_botocore_pip
-boto3>=1.20.0
-botocore>=1.23.0
+boto3>=1.25
```
Comment (gravesm) [inline requirements.txt:5]:
```suggestion
boto3>=1.25.0
```

File: changelogs/fragments/bump_boto3_botocore_versions.yml [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+  ``botocore<1.23.0`` and ``boto3<1.20.0``.
```
Comment (gravesm) [inline changelogs/fragments/bump_boto3_botocore_versions.yml:3]:
```suggestion
  ``botocore<1.28.0`` and ``boto3<1.25.0``.
```

**Review by jillr — APPROVED** [review]

**Comment by alinabuzachis** [comment]
regate

**Comment by alinabuzachis** [comment]
recheck

**Pull request merged**